Sources/TokamakStaticHTML/Resources/TokamakStyles.swift

```diff
@@ -99,14 +99,20 @@ public let tokamakStyles = """
 ._tokamak-securefield,
 ._tokamak-textfield-default,
 ._tokamak-textfield-roundedborder,
-._tokamak-picker {
+._tokamak-picker,
+._tokamak-texteditor {
   color-scheme: light dark;
 }
```

Comment:
I wonder if at this point it makes sense to add a `_tokamak-formcontrol` class for this